ambuda/queries.py

```diff
@@ -169,7 +169,39 @@ def create_user(*, username: str, email: str, raw_password: str):
     session = get_session()
     user = db.User(username=username, email=email)
     user.set_password(raw_password)
-
     session.add(user)
+    session.flush()
+
+    # Allow all users to be proofreaders
+    user_permission = db.UserPermission(
+        user_id=user.id, permission=db.UserPermissionValues.PROOFREADER.value
+    )
+    session.add(user_permission)
+
     session.commit()
     return user
+
+
+def get_user_permission(user_id: int) -> Optional[db.UserPermission]:
```

Comment:
This can be done with a SQLAlchemy relationship, see `pages` on `Project` for an example.